core/DAPS/README.md

```diff
@@ -1,16 +1,212 @@
 # Dynamic Attribute Provisioning Service (DAPS)
 
+The infrastructure component "Digital Attribute Provisioning Service" (DAPS)
+ in a given IDS ecosystem enables enrichment of identities of organisations
+ and connectors with additional attributes.
+ 
+DAPS can be understood (like the [Certification Authority](../CA/README.md), too) as building block of the construct of an
```

Comment:
Should be "as a building block", sentence is a bit hard to parse.
Maybe "Like the Certification Authority, the DAPS can be understood as a building block of an IDS Identity Provider." instead?

Reply:
Sounds better.